editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
Hm 🤔 Making a selection doesn't seem like the sort of action you take when you want to insert something. If anything, that would mean, delete the selection and insert something instead, but then I'd rather have it explicitly separate.

The inspector toggle is currently hidden when the multi-select header takes over, not sure if we want to keep something there, but both the inserter and expect would then be action tied to multi-selection and would not remove the selection on click.

> To me is seems better that clicking or moving focus anywhere outside the UI that has to do with multi-selection should deselect.